Document/0x03-Overview.md

```diff
@@ -1,61 +1,78 @@
+
+2. The [Android Testing Guide](0x05a-Platform-Overview.md) covers mobile security testing for the Android platform, including security basics, security test cases, reverse engineering techniques and prevention, and tampering techniques and prevention.
+
+3. The [iOS Testing Guide](0x06a-Platform-Overview.md) covers mobile security testing for the iOS platform, including an overview of the iOS OS, security testing, reverse engineering techniques and prevention, and tampering techniques and prevention.
+
+## How Security Personnel Should Address Mobile Security Testing
+
+Many mobile app penetration testers have a background in network and web app penetration testing, a quality that is valuable for mobile app testing. Almost every mobile app talks to a backend service, and those services are prone to the same types of attacks we are familiar with in web apps on desktop machines. Mobile apps have a smaller attack surface and therefore have more security against injection and similar attacks. Instead, the MASTG prioritizes data protection on the device and the network to increase mobile security.
 
-Many mobile app penetration testers have a background in network and web app penetration testing, a quality that is valuable for mobile app testing. Almost every mobile app talks to a backend service, and those services are prone to the same types of attacks we are familiar with in web apps on desktop machines. Mobile apps differ in that there is a smaller attack surface and therefore more security against injection and similar attacks. Instead, we must prioritize data protection on the device and the network to increase mobile security.
+## OWASP MASVS Overview: Key Areas in Mobile Application Security
 
-Let's discuss the key areas in mobile app security.
+This overview discusses how the MASVS defines and describes the key areas of mobile security:
 
-### Data Storage and Privacy (MASVS-STORAGE)
+[Data Storage and Privacy](#datastorage)
+[Cryptography](#crypto)
+[Authentication and Authorization](#auth)
+[Network Communication](#network)
+[Interaction with the Mobile Platform](#code)
+[Anti-Tampering and Anti-Reversing](#resilience)
 
-The protection of sensitive data, such as user credentials and private information, is crucial to mobile security. If an app uses operating system APIs such as local storage or inter-process communication (IPC) improperly, the app might expose sensitive data to other apps running on the same device. It may also unintentionally leak data to cloud storage, backups, or the keyboard cache. Additionally, mobile devices can be lost or stolen more easily compared to other types of devices, so it's more likely an individual can gain physical access to the device, making it easier to retrieve the data.
+### <A id="datastorage"></a> MASVS-STORAGE: Data Storage and Privacy
```

Comment:
Why these `<a>` elements? Markdown generates HTML IDs automatically

Reply:
I see now. You want to include an index. You can use the sections titles. Just put them in all lowercase and use dashes. For example:

### MASVS-STORAGE: Data Storage and Privacy

Turns into: masvs-storage-data-storage-and-privacy

Reply:
I got it now. Beg pardon.